resources/js/features/game-list/components/SuggestionReasonCell/CommonPlayersReason/CommonPlayersReason.tsx

```diff
@@ -0,0 +1,37 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuCircleDot, LuTrophy } from 'react-icons/lu';
+
+import { BaseChip } from '@/common/components/+vendor/BaseChip';
+import { GameAvatar } from '@/common/components/GameAvatar';
+
+interface CommonPlayersReasonProps {
+  relatedGame: App.Platform.Data.Game;
+  sourceGameKind: App.Platform.Services.GameSuggestions.Enums.SourceGameKind;
+}
+
+export const CommonPlayersReason: FC<CommonPlayersReasonProps> = ({
+  relatedGame,
+  sourceGameKind,
+}) => {
+  const { t } = useTranslation();
+
+  const IconComponent = sourceGameKind === 'beaten' ? LuCircleDot : LuTrophy;
+
+  return (
+    <BaseChip
+      data-testid="common-players-reason"
+      className="flex gap-1.5 whitespace-nowrap py-1 text-neutral-300 light:text-neutral-900"
+    >
+      <IconComponent className="size-[18px] lg:hidden xl:block" />
+
+      <span className="inline xl:hidden">
+        {sourceGameKind === 'beaten' ? t('Similar beats') : t('Similar masteries')}
+      </span>
+      <span className="hidden xl:inline">
+        {sourceGameKind === 'beaten' ? t('Often beaten with') : t('Often mastered with')}
```

Comment:
I realize the real estate is limited (will that be a problem with translations), but this is misleading. It could be years between the masteries. Would "Mastered by players of" be too long?

Reply:
In latest, I've stolen some space from the Title column:
![Screenshot 2025-01-11 at 4 44 19 PM](https://github.com/user-attachments/assets/36f99da2-4597-47e2-b28b-88614f2bc842)
This is an absurd example, but should demonstrate that translations ought to be okay.

I've also changed this text to "Mastered by players of" and "Beaten by players of".